yt_dlp/extractor/chzzk.py

```diff
@@ -146,19 +146,39 @@ def _real_extract(self, url):
         video_meta = self._download_json(
             f'https://api.chzzk.naver.com/service/v3/videos/{video_id}', video_id,
             note='Downloading video info', errnote='Unable to download video info')['content']
-        formats, subtitles = self._extract_mpd_formats_and_subtitles(
-            f'https://apis.naver.com/neonplayer/vodplay/v1/playback/{video_meta["videoId"]}', video_id,
-            query={
-                'key': video_meta['inKey'],
-                'env': 'real',
-                'lc': 'en_US',
-                'cpl': 'en_US',
-            }, note='Downloading video playback', errnote='Unable to download video playback')
+        video_status = video_meta.get('vodStatus')
+        video_live_date = video_meta.get('liveOpenDate')
+        if video_live_date:
+            live_status = 'was_live'
+        else:
+            live_status = 'not_live'
 
```

Comment:
- let's move `video_status` to the bottom here / to the beginning of the next code block since it's only relevant to that block
- is `liveOpenDate` useful metadata? should we return it is `release_timestamp` or something? if not, let's condense this:
```suggestion

        live_status = 'was_live' is video_meta.get('liveOpenDate') else 'not_live'
        video_status = video_meta.get('vodStatus')
```